### ISSUE TITLE
up/down connections

### ISSUE THREAD

**Issue by mlengle** [Neop/mudmap2#58]
How should I create/display those? Layers/maps do not look like designed for presenting vertical layers in a zone, I can't find how to connect rooms in adjacent layers. Using diagonal room placement for up/down is very clumsy cause by default they're treated as NE/SW exits.

**Comment by Neop** [comment]
You can add up/down connections in the "Paths / Exits" -> "Connect" -> "Select" menu. There you can select the second place (may be on a different map layer), then select "u" and "d" as the up/down exits below. Btw, you can also select "-" for a one-way exit or enter custom exit names there.

**Comment by mlengle** [comment]
@Neop but you understand that's incredibly convoluted way to build a multilayered map comparing to 4 cardinal directions that you can connect with hot keys and see without wrangling the mouse? Maps concept in the current ui is indeed far from layers, you can't overlap them with a transparency or actually align in a orthogonal way

**Comment by Neop** [comment]
I know it's not perfect, it's as good as it will be. Years ago when I created this software (mostly for fun and to practice) I decided to create the map rendering and interaction logic with 2D vector tools, which was tricky enough with only one layer shown and I remember some performance issues too. Showing multiple layers with transparency, 3D or so would require recreating big parts of the program for which I don't have time and motivation these days.
